edx_event_bus_kafka/internal/consumer.py

```diff
@@ -179,18 +192,32 @@ def reset_offsets(consumer, partitions):
             self.consumer.subscribe([full_topic], on_assign=reset_offsets)
             logger.info(f"Running consumer for {run_context!r}")
 
+            # How many errors have we seen in a row? If this climbs too high, exit with error.
+            # Any error counts, here—whether due to a polling failure or a message processing
```

Comment:
Assuming `here-whether` is a typo?

Reply:
No—it's an em-dash, but I guess those don't show up well in fixed-width fonts. :-)